compoments/report-repair/repair-problem-related-to.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import TitledRadioFieldSet from '../titledRadioFieldSet';
+
+const RepairProblemRelatedTo = ({handleChange, values, options}) => {
+  const name =  'repairProblemRelatedTo';
+  const title =  'What does your problem relate to?';
+
+  return <TitledRadioFieldSet
```

Comment:
why can't we call just call `RadioFieldSet` here?

Reply:
Agreed to remove `TitledRadioFieldSet`